frontend/package.json

```diff
@@ -18,15 +18,16 @@
     "@docsearch/react": "3.6.0",
     "@docusaurus/core": "^3.1.1",
     "@docusaurus/preset-classic": "^3.1.1",
+    "@docusaurus/module-type-aliases": "^3.1.1",
```

Comment:
Why do we need changes to the main `package.json`

Reply:
It looks like all the Docusaurus-releated packages are marked as `dependencies`, and [only `@docusarus/module-type-aliases` is marked as a `devDependency`](https://github.com/saucelabs/elemental-next/blob/4b55d9599ff64c3eb448ff6b5da8d4cbf0a77aae/frontend/package.json#L17-L30). It didn't seem intentional, so I thought I'd clean that up while I was in the area.

If you prefer to keep it as it was I can revert that?

Reply:
I reverted the change